src/main/java/org/broadinstitute/consent/http/util/gson/GsonUtil.java

```diff
@@ -0,0 +1,19 @@
+package org.broadinstitute.consent.http.util.gson;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+import java.time.Instant;
+
+public class GsonUtil {
+    public static Gson buildGson() {
+        return gsonBuilderWithAdapters().create();
+    }
+
+    public static GsonBuilder gsonBuilderWithAdapters() {
+        return new GsonBuilder()
+                .registerTypeAdapter(
+                        Instant.class,
+                        new InstantTypeAdapter());
+    }
```

Comment:
The FSO class is the first instance of us using `Instant` in an object which is converted into Json. We needed to be able to use a custom type converter because Gson doesn't know about. modern DateTime objects. Thus, I created these methods, and updated the default Gson used by consent to use this custom Gson object.

Reply:
Oooh, this is more than I expected. I'm not happy that Gson doesn't understand this. I learned recently that Gson doesn't like record classes. Might need to reflect on this for a bit.